python/tests/integration_tests/test_terra_util.py

```diff
@@ -0,0 +1,179 @@
+import pytest
+import requests
+import os
+import re
+
+from python.utils.request_util import RunRequest
+from python.utils.terra_utils.terra_util import TerraWorkspace, TerraGroups, MEMBER
+from python.utils.terra_utils.terra_workflow_configs import WorkflowConfigs
+from python.utils.token_util import Token
+
+INTEGRATION_TEST_TERRA_BILLING_PROJECT = "ops-integration-billing"
+INTEGRATION_TEST_TERRA_WORKSPACE_NAME = "ops-integration-test-workspace"
+INTEGRATION_TEST_GROUP_NAME = "ops-integration-test-group"
+
+auth_token = Token(cloud="gcp")
```

Comment:
Could we import GCP from util init to keep consisitent?

Reply:
Good catch, updated!